packages/dds/tree/src/test/shared-tree/fuzz/composeVsIndividual.fuzz.spec.ts

```diff
@@ -90,16 +99,31 @@ const fuzzComposedVsIndividualReducer = combineReducersAsync<
 		assert(branch !== undefined);
 		const nodeSchema = branch.currentSchema;
 
-		const nodeTypes: TreeNodeSchemaIdentifier[] = [];
+		const nodeTypes: string[] = [];
 		for (const leafNodeSchema of nodeSchema.info.optionalChild.allowedTypeSet) {
 			if (typeof leafNodeSchema !== "string") {
-				nodeTypes.push(leafNodeSchema.name);
+				nodeTypes.push(leafNodeSchema.identifier);
 			}
 		}
-		nodeTypes.push(brand(operation.operation.type));
-		const { leafNodeSchemas, library } = generateLeafNodeSchemas(nodeTypes);
-		const newSchema = createTreeViewSchema(leafNodeSchemas, library);
-		branch.checkout.updateSchema(intoStoredSchema(newSchema));
+		nodeTypes.push(operation.contents.type);
+		const leafNodeSchemas = generateLeafNodeSchemas(nodeTypes);
+		const newSchema = createTreeViewSchema(leafNodeSchemas);
+		const newFork = branch.checkout.fork();
+		branch.dispose();
```

Comment:
commenting the body of this method (it isn't clear why you're disposing the local branch and using a fork, or why we are assigning a node schema to the view) would be helpful

Reply:
After chatting with @CraigMacomber and @yann-achard-MS we decided to remove schemaOps from this test, as there was no good way of continuing the transaction after applying a schemaOp (since we create a new view from the fork when applying the schemaOp).